src/lib/Pjax/PjaxResponseMainContentMapper.php

```diff
@@ -50,14 +62,46 @@ public function map($response)
         $xpath = new \DOMXPath($doc);
         $title = $xpath->query('//div[@data-name="title"]')[0]->nodeValue;
         $content = $this->innerHtml($xpath->query('//div[@data-name="html"]')[0]);
+        $notificationElements = $xpath->query('//ul[@data-name="notification"]/li');
 
         $this->app->setConfig([
             'title' => $title,
             'toolbars' => ['discovery' => 1],
             'mainContent' => ['result' => '<ez-server-side-content>' . $content . '</ez-server--side-content>'],
+            'notifications' => $this->buildNotifications($notificationElements),
         ]);
     }
 
+    /**
+     * Builds an array of notifications for Hybrid Platform UI app from PJAX
+     * Notification HTML elements.
+     *
+     * @param \DOMNodeList $elements
+     * @return array
+     */
+    private function buildNotifications(\DOMNodeList $elements)
+    {
+        $notifications = [];
+
+        foreach ($elements as $element) {
```

Comment:
Overall, a `notificationMapper` would make the spec much easier to write. I'm just not sure it's worth doing it now. What do you think @dpobel ?